fe/fe-core/src/main/java/org/apache/doris/datasource/InternalCatalog.java

```diff
@@ -807,67 +807,70 @@ public void replayRenameDatabase(String dbName, String newDbName) {
 
     // Drop table
     public void dropTable(DropTableStmt stmt) throws DdlException {
-        String dbName = stmt.getDbName();
-        String tableName = stmt.getTableName();
+        for (TableName tbl : stmt.getTableNames()) {
+            String dbName = tbl.getDb();
+            String tableName = tbl.getTbl();
```

Comment:
We'd better check before dropping and dropping multi tables in a transaction if possible.